develop/addons/components/genericsetup.rst

```diff
@@ -717,203 +742,1393 @@ Example:
 
-        <plugin-type id="IUserAdderPlugin" title="user_adder"
-                description="User Adder plugins allow the Pluggable Auth Service to create users."
-                interface="Products.PluggableAuthService.interfaces.plugins.IUserAdderPlugin">
-        </plugin-type>
-    </plugin-registry>
+Code is in ``plone.app.contentrules.exportimport.rules``.
 
 
-Best Practices
---------------
+Content Generation
+------------------
 
-When importing items such as property sheets, make sure not to
-override other profile settings by setting the purge attribute to False.
-This will add the items listed to the property instead of resetting the
-property. Example:
+Code is in ``Products.GenericSetup.content``:
 
-.. code-block:: xml
+.. automodule:: Products.GenericSetup.content
+ :members: FolderishExporterImporter
 
-    <property name="metaTypesNotToList" type="lines" purge="False">
-      <element value="File"/>
-      <element value="Image"/>
-    </property>
 
-Only use the configuration that you need. When you export your site's
-configuration, it will include things that you don't need. For example,
-if you needed to change only the 'Allow anonymous to view about'
-property, this is what your propertiestool.xml would look like:
+controlpanel.xml
+----------------
 
 .. code-block:: xml
 
     <?xml version="1.0"?>
-    <object name="portal_properties" meta_type="Plone Properties Tool">
-      <object name="site_properties" meta_type="Plone Property Sheet">
-        <property name="allowAnonymousViewAbout" type="boolean">True</property>
-      </object>
-    </object
+    <object
+        name="portal_controlpanel"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="plone.app.caching">
+
+      <configlet
+          title="Caching"
+          action_id="plone.app.caching"
+          appId="plone.app.caching"
+          category="plone-advanced"
+          condition_expr=""
+          icon_expr="string:$portal_url/++resource++plone.app.caching.gif"
+          url_expr="string:${portal_url}/@@caching-controlpanel"
+          visible="True"
+          i18n:attributes="title">
+        <permission>Manage portal</permission>
+      </configlet>
+
+    </object>
+
+This creates an action in the Site Setup control panel in Plone.
+Actions are bundled in categories.
+The only categories supported in standard Plone 5 are:
+
+- ``Member`` (My Preferences)
+- ``Plone`` (Plone Configuration)
+- ``plone-advanced`` (Advanced)
+- ``plone-content`` (Content)
+- ``plone-general`` (General)
+- ``plone-security`` (Security)
+- ``plone-users`` (Users)
+- ``Products`` (Add-on Configuration)
+
+Any other categories are not displayed in the overview control panel.
+Note that in Plone 4, only ``Member``, ``Plone`` and ``Products`` were supported.
+For add-ons, the category ``Products`` is recommended.
+
+The ``action_id`` must be unique over all categories.
+
+Only one permission is allowed.
+
+Uninstall example:
 
```

Comment:
Does the info for Plone 4 has to be here ? We have own docs for Plone 4, so if it has to stay here, we should use .. versionadded:: version